packages/lodestar/src/api/impl/beacon/blocks/index.ts

```diff
@@ -1,4 +1,6 @@
 import {routes} from "@chainsafe/lodestar-api";
+
+// eslint-disable-next-line no-restricted-imports
```

Comment:
Why is this line necessary?

Reply:
removed, may be got included with some rebasing artifacts